corehq/apps/receiverwrapper/util.py

```diff
@@ -87,20 +87,42 @@ def get_version_from_appversion_text(appversion_text):
     ...     'CommCare Version 2.4. Build 10083, built on: March-12-2013'
     ... )
     19
-
     """
-
     patterns = [
         r' #(\d+) ',
         'b\[(\d+)\]',
         r'App v(\d+).',
     ]
-    if appversion_text:
+    version_string = _first_group_match(appversion_text, patterns)
+    if version_string:
+        return int(version_string)
+
+
+def get_commcare_version_from_appversion_text(appversion_text):
```

Comment:
i have the same thought, was just following the example from `get_version_from_appversion_text` above